Saule/Serialization/ApiError.cs

```diff
@@ -50,6 +73,26 @@ private static string GetRecursiveExceptionMessage(HttpError ex)
             }
 
             return msg;
+        }
+
+        private ExceptionRenderer GetRenderer()
```

Comment:
Let's think of a more flexible way to pick a renderer. We'd want people to supply one somehow. Perhaps using a factory in the configuration object?